Author: Mario Lang <mlang@debian.org>
Description: Adds new magic for SuperCollider 3 Synth Definition File Format (Closes: #284803).

diff -Naurp file.orig/magic/Magdir/audio file/magic/Magdir/audio
--- file.orig/magic/Magdir/audio	2008-01-12 17:39:01.000000000 +0000
+++ file/magic/Magdir/audio	2008-01-12 17:38:49.000000000 +0000
@@ -576,3 +576,8 @@
 # URL:  http://filext.com/detaillist.php?extdetail=AMR
 # From: Russell Coker <russell@coker.com.au>
 0	string	#!AMR		Adaptive Multi-Rate Codec (GSM telephony)
+
+# Type: SuperCollider 3 Synth Definition File Format
+# From: Mario Lang <mlang@debian.org>
+0	string	SCgf	SuperCollider3 Synth Definition file,
+>4	belong	x	version %d
